common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Relations.scala

```diff
@@ -64,6 +64,17 @@ object Relation {
       numDescendents = numDescendents,
     )
 
+  /**
+    * A Relation created with just a String is a Series.  The string is the title.
+    * A Series is not an entity in its own right, so does not have an id of its own.
+    *
+    * Practically, a Series does not exist in a hierarchy, so does not have depth or
+    * children or descendants, even though (obviously) there are objects that are "partOf"
+    * the series.
+    */
+  def apply(series: String): Relation =
+    Relation(None, Some(series), None, WorkType.Series, 0, 0, 0)
```

Comment:
Couple of notes:

1. Could we pull this into a dedicated object so it's clear what this is doing elsewhere in the codebase? e.g.

    ```scala
    object SeriesRelation {
      def apply(name: String): Relation = ...
    }

    // elsewhere
    val series = SeriesRelation(name = "Early Printed Books")
    ```

2. Let's add some keyword arguments so we know what these `None` and `0` values mean.